docs/reference/cat/nodes.asciidoc

```diff
@@ -315,6 +315,9 @@ Time spent in suggest, such as `0`.
 `suggest.total`, `suto`, `suggestTotal`::
 Number of suggest operations, such as `0`.
 
+`shard_stats.total_count`, `sstc`, `shardStatsTotalCount`::
+Number of shards assigned.
```

Comment:
We already have a disk metrics here as well. 
I think it would be great to also expose ingest/disk forecasts here (computed per node from the cluster state) to easier torubleshoot the cluster balance.

Reply:
Sure, also perhaps in `GET _cat/allocation`. Although mostly I use nodes stats (plus some `jq`) for this kind of thing so as long as it's there I don't mind so much.